prime-router/metadata/HL7/catchall/hl7/segments/ORC/OBRExtension.yml

```diff
@@ -34,6 +34,70 @@ extension:
           valueOf: datatypes/EI/Identifier
           expressionType: resource
           specs: OBR.3
+    - expressionType: nested
+      vars:
+        obr8: OBR.8
+      condition: $obr8 NOT_NULL
+      generateList: true
+      expressionsMap:
+        url:
+          type: STRING
+          value: OBR.7, OBR.8
```

Comment:
Since the extensions are for our reference only I think this would be simpler keeping OBR.7 and OBR.8 as separate elements to better reflect how they were sent to us in the original HL7. We should also consider (as a separate ticket) a unified datetime HL7 to FHIR mapping so that we can have a consistent way of dealing with them rather than their current ad hoc implementations, to better align with the DTMorDT mapping on the FHIR to HL7 end.

Reply:
Separated out OBR.7 and OBR.8 when they are populated in OBRExtension. Both are now following the `valueDateTime`/`_valueDateTime` pattern: https://github.com/CDCgov/prime-reportstream/pull/16539/commits/78009ee9d98430a4524d988a5b95072622eaf9a4